pkg/collector/python/datadog_agent.go

```diff
@@ -609,3 +610,25 @@ func ObfuscateMongoDBString(cmd *C.char, errResult **C.char) *C.char {
 	// memory will be freed by caller
 	return TrackedCString(obfuscatedMongoDBString)
 }
+
+// SendAgentTelemetry records a telemetry data point for a Python integration
+//
+//export SendAgentTelemetry
+func SendAgentTelemetry(checkName *C.char, metricName *C.char, metricValue *C.float) {
+	// telemetryType and telemetryValue are optional
+	goCheckName := C.GoString(checkName)
+	goMetricName := C.GoString(metricName)
+	goMetricValue := float64(*metricValue)
+
+	switch goCheckName {
+	case "postgres":
+		switch goMetricName {
+		case "activity_latency":
+			postgres.TlmPostgresActivityLatency.Observe(goMetricValue)
+		default:
+			log.Warnf("SendAgentTelemetry: unsupported check name %s", goCheckName)
```

Comment:
This is interesting. Indeed if a check will be upgraded but python not (yet) it will generate error., Will be metric types/content  be that different per DB. I mean can't reuse metric like activity_latency with different tag. I do not know what is better even if they largely overlaps.

Reply:
It's mostly a debugging tool against typos since we don't have any way to have type safe enums across the C bridge as far as I know. It's true there will be a lot of overlap across DB checks but it won't be 100% and being explicit makes it easier to track down errors